apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -153,3 +157,13 @@ var (
 		},
 	}
 )
+
+func GetKubernetesServicesOptions(version string) KubernetesServicesOptions {
```

Comment:
do you think we could define this function in RKE? lets keep type for object definition, not helper functions